adder.js

```diff
@@ -1 +1,15 @@
-// code here
+function add(num1, num2) {
+  // num1 = parseInt(prompt("something"));
+  // num2 = parseInt(prompt("something"));
+  return num1 + num2;
+}
+function print(num) {
```

Comment:
to clarify, the function is essentially printing a string and is not restricted only for numbers. If you had bigger program, you could reuse better if you pass a parameter called e.g. _message_
`function print(message) {}`